### ISSUE TITLE
"image" as the label in "image alternatives" 2.1 Image and media alternatives Developing Guideline 2.1.1 Image alternatives

### ISSUE THREAD

**Issue by brett-oppegaard** [w3c/wcag3#371]
While technically accurate, to a degree, the "image" label in WCAG is severely outdated and should be updated. 

No. 1, when most people think about "image," they do not think about maps, tables, charts, data visualizations, etc., at least in any significant depth, which creates major accessibility issues for audience members who can't see a map on a website and get as the "image description" something like "Yellowstone National Park map.jpg," which, of course, is totally unusable even for cognitive mapping exercises, like where is Old Faithful in relation to Yellowstone Falls?

I suggest aligning with the use of "Audio Description" by [The American Council of the Blind](https://adp.acb.org/), [The American Foundation of the Blind](https://www.afb.org/aw/22/6/17586), the [Royal National Institute of the Blind](https://www.rnib.org.uk/living-with-sight-loss/assistive-aids-and-technology/tv-audio-and-gaming/audio-description-ad/), the [World Blind Union](https://www.pathstoliteracy.org/resource/results-first-worldwide-survey-audio-description-activity/), and dozens of other associations of blind people around the world, who use that term more often than others (like image description, verbal description, etc.), plus associations of public attractions, such as museums (i.e., [The American Alliance of Museums](https://www.aam-us.org/2019/08/28/partnering-with-the-blind-to-make-exhibits-more-accessible/)), libraries ([Washington State University Libraries](https://crln.acrl.org/index.php/crlnews/rt/printerFriendly/26803/34733)), national parks ([U.S. National Park Service](https://www.nps.gov/subjects/hfc/upload/DOI_CreatveCommAD22MAR2018.pdf)), etc.  ... I have documented about 20 different terms for this same basic process of audiovisual translation/remediation, and many organizations, including of blind people, think first of video/TV as the application for AD. Yet the vast majority of people working in this field academically (in the humanities) use AD more generally (computer scientists almost entirely use "image" description as the term, more on that below). Practitioners might just refer to it as "description," but there is a long history of its development as a very specific label in the United States (per this link, https://journals.sagepub.com/doi/abs/10.1177/0145482X221116903), which originated in the United States and then spread to the rest of the world. 

2. In the humanities, in academia, scholars generally split their interests into two camps: A. Dynamic Audio Description, for videos, TV, sporting events, theater, dance, etc., and B. Static Audio Description for photographs, illustrations, maps, tables, charts, etc. I'm in Camp B. ... It's one of those situations where no matter what term gets chosen, other people are going to be using different labels. So we pick the most prominent one among the biggest audience of listeners, aligning with people who are blind or with low-vision (focusing on the primary audience for the description, rather than the primary producers of the descriptions). Also, if you want to dig deeper into this rabbit hole, I recommend checking out the [ARSAD conference](https://webs.uab.cat/arsad/), which is the only major AD conference in the world for scholars. 

3. In legal terminology, the issue gets more vague, with references to "descriptive aids" and "alternative texts." But, in short, making media accessible to the general public — regardless of a person’s sensory abilities — is a legal mandate that has been in place and reinforced in the U.S. for decades, starting with [Section 504 and Section 508 of the Rehabilitation Act of 1973](https://webaim.org/articles/laws/usa/rehab), and expanded by a variety of other key pieces of legislation or rulings, including the [Americans with Disabilities Act in 1990](https://www.ada.gov/), the [2010 ADA Standards for Accessible Design](https://www.ada.gov/law-and-regs/design-standards/2010-stds/), and most recently, the final [Department of Justice ruling](https://www.ada.gov/resources/2024-03-08-web-rule/) on regulations of Title II of the ADA. There is a consistency of intent among the laws but not labels. WCAG should help to set this standard, not to confuse it even more by loosely using a term that is too narrow. 

4. I could go point by point why I think AD is the best term for this subject matter (AD, by the way, is both a product and a process, making it even more confusing!). But I'd say the two main competitors for the label are Audio Description and Image Description (which the computer science / tech folks use). In short, though, Image Description, for me, and probably for all of the organizations that have opted instead for AD, is too limiting of a term, because most people would not call a map, a table, or a chart an image, even if it is presented as an "image," and beyond that, we often describe other static media, such as waysides signs (in national parks, for the content on the signs as well as the physical environment around them), plus landscapes/views, on their own, exhibits, objects/artifacts, such as statues, [university campuses and buildings (inside and out)](https://unidescription.org/account/project/export/324), classroom materials, and so on. Image Description just doesn't work very well in those cases.

Hope that helps!

Commenting on:

2.1 Image and media alternativesDeveloping
Guideline 2.1.1 Image alternatives

Foundational requirement: Equivalent text alternativeDeveloping
Equivalent

text alternatives
are available for [images](https://www.w3.org/TR/wcag-3.0/#dfn-image) that convey [content](https://www.w3.org/TR/wcag-3.0/#dfn-content).
